core/dispatcher/src/main/scala/whisk/core/dispatcher/Dispatcher.scala

```diff
@@ -191,8 +195,6 @@ object Dispatcher extends Logging {
     def requiredProperties =
         Map(servicePort -> 8080.toString()) ++ kafkaHost
 
-    val executionContext = ExecutionContextFactory.makeCachedThreadPoolExecutionContext()
```

Comment:
I assume because in theory you may need an unbounded number of concurrent executors? I can amend the PR to keep a cached pool and make it the underlying context of the `ActorSystem` (just like we have now), but still have a single context (like this PR is introducing).